custom-service-adapter/README.md

```diff
@@ -0,0 +1,19 @@
+# Custom Service Adatper with Knot.x
+
+In this tutorial we create a page: [http://localhost:8092/html/books.html](http://localhost:8092/html/books.html) books from database.
+Database access is not available via WebAPI thus we have 2 options:
+
+1. integrate with data source implementing WebAPI layer
+2. write simple [Knot.x Service Adapter](https://github.com/Cognifide/knotx/wiki/ServiceAdapter)
+
+Before running this example you will need to setup HSQL database.
+You may read how to do this in [this tutorial](http://o7planning.org/en/10287/installing-and-configuring-hsqldb-database).
+To create database use script provided in `db` folder of this tutorial root.
```

Comment:
To create tables with data use the script provided in `db` folder of this tutorial